### ISSUE TITLE
Python 3.9 available on Travis

### ISSUE THREAD

**Issue by jugmac00** [zopefoundation/meta#37]
Finally...

https://travis-ci.community/t/python-3-9-0-build/10091

**Comment by icemac** [comment]
Thank you for this info. We do not have the infrastructure to automatically switch from `3.9-dev` to `3.9` so we have to do it manually when encountering a dev version.

I am going to close this issue as I see no actionable items here.
Feel free to reopen if you disagree.

**Comment by jugmac00** [comment]
At first - yes, this was only informational, but when you said we have no infrastructure for mass changes, I remembered that I evaluated `all-repos` by @asottile a year ago, but could not wrap my head around the autofixer process, so I had another look today... and what the f... looks like I am creating tons of PRs currently, automatically!

e.g. https://github.com/zopefoundation/zope.sqlalchemy/pull/50

I will try to find some time to write a blog post.
